target/linux/ipq806x/files-5.4/arch/arm/boot/dts/qcom-ipq8065-r7800.dts

```diff
@@ -1,83 +1,25 @@
-#include "qcom-ipq8065.dtsi"
-
-#include <dt-bindings/input/input.h>
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
Changing/adding a license to an existing file requires consent of all previous contributers.

Reply:
Should we really add a license to this? Isn't a bit overkill ?

Reply:
Recently, DTS files have been required to have a license to be merged. However, this is applied to "new" files only, as contacting all people having touched an old file may be painful. So, for the case at hand, it might indeed by easiest to drop (i.e. not add) the license for this particular file.

Reply:
can we treat qcom-ipq8065-r7800.dts as "new file" as 80% if it's content was changed and were generalized into dtsi file?

Reply:
No, but if you follow that path you would have to ask everybody for the DTSI as well.

Reply:
@Ansuel : as you are initial author if file https://github.com/openwrt/openwrt/commit/9a1ec4d3f4fca05275be28a6927b08c6cd43d97d#diff-7e386849ecbd95ada049a3e851054256 . Is it possible that you can consider to apply GPL-2.0-or-later OR MIT license for mentioned file?

Reply:
I've removed conflicting line in r7800 dts file so it remains without license.